minimesos/src/main/groovy/com/containersol/minimesos/config/ClusterConfig.groovy

```diff
@@ -53,4 +56,14 @@ class ClusterConfig extends GroovyBlock {
         delegateTo(marathon, cl)
     }
 
+    void setLoggingLevel(String loggingLevel) {
+        if (!StringUtils.equalsIgnoreCase(loggingLevel, "WARNING") && !StringUtils.equalsIgnoreCase(loggingLevel, "INFO") && !StringUtils.equalsIgnoreCase(loggingLevel, "ERROR")) {
+            throw new RuntimeException("Property 'loggingLevel' can only have the values INFO, WARNING or ERROR")
+        }
+        this.loggingLevel = loggingLevel
+    }
```

Comment:
This one is very different now from implementation in `MesosContainerConfig`
```
+    public void setLoggingLevel(String loggingLevel) {
 +        this.loggingLevel = loggingLevel.toUpperCase()
 +    }
```

`WARN`, not `WARNING` is commonly used logging level. Should we use it?